src/components/navigation/Navbar.scss

```diff
@@ -0,0 +1,103 @@
+@import "../../constants.scss";
+
+.navbar {
+  display: block;
+  position: fixed;
+  top: 20px;
+  left: 20px;
+  height:100%;
+  z-index: 1;
+  -webkit-user-select: none;
+  user-select: none;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: white;
+  transition: color 0.3s, underline 0.3s;
+}
+
+.navbar a:hover {
+  color: black;
+  text-decoration: underline;
+}
+
+/*
+ * Just a quick hamburger
+ */
+.navbar span {
+  display: block;
+  width: 33px;
+  height: 4px;
+  margin-bottom: 5px;
+  position: relative;  
+  background: $secondary;
+  border-radius: 3px; 
+  z-index: 1;
+  transform-origin: 4px 0px;
+  transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+              background 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+              opacity 0.55s ease;
+}
+
+.navbar span:first-child {
+  transform-origin: 0% 0%;
+}
+
+.navbar span:nth-last-child(2) {
+  transform-origin: 0% 100%;
+}
+
+.burgerexpanded {
```

Comment:
In BEM, "expanded" is a modifier (the "M" in BEM) so we'd probably call this class:
`.navbar__burger--expanded`

Reply:
done, thanks